src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,72 @@
+
+  public Intake() {
+    fieldsTable.update();
+  }
+
+  @Override
+  public void periodic() {
+  }
+
+  public void setSpeedIntake(double intakeSpeed) {
+    intakeSpeed = rollersSpeedLimiter.calculate(intakeSpeed);
+    io.setRollerSpeed(MathUtil.clamp(
+      intakeSpeed,
+      -ROLLERS_LIMIT_ACCELERATION_VOLTEG_PER_SECOND,
+       ROLLERS_LIMIT_ACCELERATION_VOLTEG_PER_SECOND));
+  }
+
+  public void setAngleIntake(double jointSpeed) { 
+    jointSpeed = wristSpeedLimiter.calculate(jointSpeed);                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
+    io.setWristSpeed(MathUtil.clamp(
+      jointSpeed,
+      -WRIST_LIMIT_ACCELERATION_VOLTEG_PER_SECOND,
+       WRIST_LIMIT_ACCELERATION_VOLTEG_PER_SECOND));
+  }
```

Comment:
The clamps shouldn't use the acceleration limit constants, but separate voltage limit constants. the acceleration limit is done by the `SlewRateLimiter`